### ISSUE TITLE
Synchronisation issue

### ISSUE THREAD

**Issue by creepy0815** [samphonic/MoppyClassic#184]
This issue occurs on my setup when I assign channel 10 to be played by the MSWavetable. Any ideas?

**Comment by samphonic** [comment]
Can you describe the issue you are experiencing?

**Comment by creepy0815** [comment]
The drummtrack runs behind the floppys a few beats. It holds the tempo perfectly and also does the changes of tempo properly though.

**Comment by Renarix** [comment]
It is delayed i guess

**Comment by samphonic** [comment]
Yeah, my best guess is that MSWavetable might be buffering or otherwise slow to react to the events... which is weird, I would expect it to be pretty real-time.  Unfortunately since I haven't experienced this myself, I don't have any pointers for a solution.
